packages/open-api-gateway/src/project/spec/open-api-spec-project.ts

```diff
@@ -0,0 +1,69 @@
+/*********************************************************************************************************************
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ******************************************************************************************************************** */
+
+import * as path from "path";
+import { Project, SampleFile, ProjectOptions } from "projen";
+import { ParsedSpec } from "./components/parsed-spec";
+
+/**
+ * Project containing the OpenAPI spec, and a parsed spec for use by the CDK construct
+ */
+export class OpenApiSpecProject extends Project {
+  public readonly specPath: string;
+  public readonly parsedSpecPath: string;
+
+  public readonly specDir: string = "spec";
```

Comment:
Should these all be configurable?

Reply:
Yeah I think that'd be nicer, will update :)